### DISCUSSION TITLE
Trailing / on breadcrumb links

### DISCUSSION THREAD

**Discussion by RonBPalmer** [felix-berlin/astro-breadcrumbs#180] [General]
I need the links generated by the breadcrumbs tool to have trailing slashes to avoid redirects. How can I accomplish this?

This is an example:
```
<li class="c-breadcrumbs__crumb">
	<a
		href="/about-us-pages/about-ffc"
		class="c-breadcrumbs__link is-current"
		aria-current="location"
	>
		about ffc
	</a>
</li>
```

**Comment by felix-berlin** [comment]
Hey @RonBPalmer, i will look into this later this weekend.

 For the automatic mode I surely need to adjust the component.

Right now you can achieve this via custom "crumbs" (manually define the crumbs).

**Comment by felix-berlin** [comment]
Starting from v1.9.0 you can add trailing slashes. See docs --> https://docs.astro-breadcrumbs.kasimir.dev/configuration/properties/#trailingSlash
